packages/@storefront/flux-capacitor/test/unit/core/requests/utils.ts

```diff
@@ -35,11 +35,15 @@ suite('requests helpers', ({ expect, stub, spy }) => {
     const query = 'dress';
     const collection = 'alternate';
     const refinements = [{ a: 'b' }, { c: 'd' }];
+    let state;
+    let sessionId;
     let sortSelector: sinon.SinonStub;
     let requestSortAdapter: sinon.SinonStub;
     let pastPurchaseBiasingAdapter: sinon.SinonStub;
 
     beforeEach(() => {
+      state = <any>{ session: { sessionId: 'foo' } };
+      sessionId = <any>RequestHelpers.extractSessionId(state);
```

Comment:
You may also want to consider stubbing `extractSessionId` instead of assuming the presence of `sessionId` in state, although you could argue that `extractSessionId` is a helper function and should be okay not being stubbed.

Reply:
I see, I will make these changes. Resolving conversation.